spec/categorical_spec.rb

```diff
@@ -0,0 +1,102 @@
+describe Daru::Vector do
+  context "initialize" do
+    let(:dv) { Daru::Vector.new [:a, 1, :a, 1, :c], type: :category }
+    subject { dv }
+    
+    it { is_expected.to be_a Daru::Vector }
+    its(:size) { is_expected.to eq 5 }
+    its(:type) { is_expected.to eq :category }
```

Comment:
Ok. Just want to confirm from @v0dro about this because he said once:

> I know I said :categorical on the listserv, but now that I think of it, we should be consistent with other tools and use :category.